src/cdomains/floatDomain.ml

```diff
@@ -36,13 +36,17 @@ module type FloatDomainBase = sig
   include FloatArith with type t := t
 
   val of_const : float -> t
+  val of_string : string -> t
   val of_int: IntDomain.IntDomTuple.t -> t
   val cast_to: Cil.ikind -> t -> IntDomain.IntDomTuple.t
 end
+
 module FloatInterval = struct
   include Printable.Std (* for default invariant, tag and relift *)
   type t = (float * float) option [@@deriving eq, ord, to_yojson]
 
+  module F32 = CFloat
```

Comment:
I don't really get this. `F32` is never used. Is the idea here simply that one can replace F32 by F64 if needed?